docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -351,6 +353,30 @@ runner.reporter(['spec', {
 }]);
 ```
 
+#### Specifying a Custom Reporter
+
+You can implement a [custom reporter](../../extending-testcafe/reporter-plugin/README.md) in the code that launches tests. Use this approach if you do not want to publish a reporter plugin.
```

Comment:
> Use this approach if you do not want to publish a reporter plugin.

I propose to remove this sentence.
You can use this approach not only if you not want to publish a reporter. In my opinion, it's more convenient to develop a reporter plugin in this way instead of the official way - https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/#implementing-the-reporter.

Reply:
Maybe we'd better add your point, not remove the sentence? Like, say that it's also good if you need your reporter just for one project, or something like that?

Reply:
Ok, let's do so.